js/liverail.js

```diff
@@ -29,7 +30,12 @@ OO.Ads.manager(() => {
     AD_LOG: 'AdLog',
   };
 
-  const Liverail = function () {
+  /**
+   * @class Liverail
+   * @classDesc The Liverail class.
+   * @constructor
+   */
+  const Liverail = function (...args) {
```

Comment:
You try to use these arguments instead of arguments in each particular function. Please, use destructure in functions, not in the constructor